langchain/src/agents/tests/create_agent_functions.test.ts

```diff
@@ -0,0 +1,98 @@
+import { test, expect } from "@jest/globals";
+import { ChatOpenAI, OpenAI } from "@langchain/openai";
+import type {
+  ChatPromptTemplate,
+  PromptTemplate,
+} from "@langchain/core/prompts";
+import { pull } from "../../hub.js";
+import {
+  createOpenAIFunctionsAgent,
+  createReactAgent,
+  createStructuredChatAgent,
+  createXmlAgent,
+} from "../index.js";
+import { ChatAnthropic } from "../../chat_models/anthropic.js";
+import { Calculator } from "../../tools/calculator.js";
+
+const tools = [new Calculator()];
+
+test("createStructuredChatAgent works", async () => {
```

Comment:
This is technically an integration test as it uses outside resources (hub, OpenAI) - I think fine to just show that you can initialize it at all with an arbitrary prompt and a FakeLLM.